### ISSUE TITLE
Docker not working

### ISSUE THREAD

**Issue by funnelcloudservices** [TeamPiped/Piped#700]
I've tried a few times on different systems thinking I was doing something wrong but I can't seem to get this fired up on Docker.  After I do the build I get "ERROR: The Compose file './docker-compose.yml' is invalid because:"

I checked the yaml file in a tester and it passes but then I ran out of talent.  It seems to be chocking on the last two lines 

```
Unsupported config option for volumes: 'caddy_data'
Unsupported config option for services: 'piped'
```
Is it me or just a few incorrect spaces accidentally left in there?

**Comment by FireMasterK** [comment]
Please use the latest docker-compose version, this would resolve your issue: https://docs.docker.com/compose/install/

I'm not sure how you got around the version check at https://github.com/TeamPiped/Piped-Docker/blob/127f4b3625af22eaf37fe2c6b4b9c052f3ec5aa7/configure-instance.sh#L7-L14, what docker-compose version do you currently have?

**Comment by funnelcloudservices** [comment]
I see what I did wrong.  I just did an apt install git, docker and docker-compose.  I might have forgotten to do an apt update too.   Derp.  I used [this guide](https://support.netfoundry.io/hc/en-us/articles/360057865692-Installing-Docker-and-docker-compose-for-Ubuntu-20-04) if anybody else is reading this and was as inept as me.  

All installed well.  Now I just need to figure out how to run it all through my existing Nginx Proxy Manager.  Thanks for your help!
